protocols/database_health_check.py

```
@@ -3,81 +3,95 @@
+
         # Check table count
-        cursor.execute("""
-            SELECT count(*) 
-            FROM information_schema.tables 
+        cursor.execute(
+            """
+            SELECT count(*)
+            FROM information_schema.tables
             WHERE table_schema = 'public';
-        """)
+        """
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)